docs/templating/twig_function_reference/site_context_twig_functions.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Site context Twig function determines if given location is site context-aware.
+edition: experience
+page_type: reference
+---
+
+# Site context Twig functions
+
+To determine if given location is site context-aware, you can use site context [Twig test](https://twig.symfony.com/doc/3.x/tests/index.html).
+
+### `ibexa_site_context_aware()`
+
+`ibexa_site_context_aware()` checks whether a given location is site context-aware.
```

Comment:
To be honest I'm not sure what 'being site context-aware" means.

Is it:
Not excluded from Site Context using the `ibexa.system.<scope>.site_context.excluded_paths` configuration?

Reply:
That's correct. If we don't have any doc explaining Site Context already, I think it's a valid point. Otherwise we should maybe link it here for further reference.